packages/api-demo/package.json

```diff
@@ -6,7 +6,7 @@
   "private": true,
   "scripts": {
     "serve:no-browser": "npm run serve -- --no-browser",
-    "serve": "live-server --mount=/:src --mount=/scripts/containerjs-api-bundle:node_modules/containerjs-api-bundle/build/containerjs-bundle.js --mount=/resources/notification.html:node_modules/containerjs-api-bundle/build/notification.html",
+    "serve": "live-server --mount=/:src --mount=/scripts/containerjs-api:node_modules/containerjs-api-browser/build/dist/containerjs-api.js --mount=/resources/notification.html:node_modules/containerjs-api-bundle/build/notification.html",
```

Comment:
I think we can remove the second part of the line that mounts the `notification.html` file (which did come from the bundle but won't any more).